config.py

```diff
@@ -182,7 +182,7 @@ def build_config(_config_file):
                         'SOFTWARE_ID': bytes(config.get(section, 'SOFTWARE_ID').ljust(40)[:40], 'utf-8'),
                         'PACKAGE_ID': bytes(config.get(section, 'PACKAGE_ID').ljust(40)[:40], 'utf-8'),
                         'GROUP_HANGTIME': config.getint(section, 'GROUP_HANGTIME'),
-                        'OPTIONS': b''.join([b'Type=HBlink;', bytes(config.get(section, 'OPTIONS'), 'utf-8')]),
+                        'OPTIONS': bytes(config.get(section, 'OPTIONS'), 'utf-8'),
```

Comment:
We're going to leave this alone. I haven't confirmed what side effects this option has when connecting to a master, but it's something Cort had put into the code in March and I am going to leave it there until I know if it has any negative side effects.